glean_parser/translation_options.py

```diff
@@ -0,0 +1,51 @@
+import pydoc
+
+
+def translate_options(ctx, param, value):
+    text = """Target language options for Translate function
+
+These are backend specific and optional, provide as key:value
+
+Rust:
+- not used
+Swift:
+- namespace: The namespace to generate metrics in
+- glean_namespace: The namespace to import Glean from
+- allow_reserved: When True, this is a Glean-internal build
+- with_buildinfo: If "true" the `GleanBuildInfo` is generated.
+        Otherwise generation of that file is skipped.
+        Defaults to "true".
+- build_date: If set to `0` a static unix epoch time will be used.
+        If set to a ISO8601 datetime string (e.g. `2022-01-03T17:30:00`)
+        it will use that date.
+        Other values will throw an error.
+        If not set it will use the current date & time.
```

Comment:
```suggestion
- `namespace`: The namespace to generate metrics in
- `glean_namespace`: The namespace to import Glean from
- `allow_reserved`: When True, this is a Glean-internal build
- `with_buildinfo`: If "true" the `GleanBuildInfo` is generated.
        Otherwise generation of that file is skipped.
        Defaults to "true".
- `build_date`: If set to `0` a static unix epoch time will be used.
        If set to a ISO8601 datetime string (e.g. `2022-01-03T17:30:00`)
        it will use that date.
        Other values will throw an error.
        If not set it will use the current date & time.
```